templates/mdai-operator.yaml

```diff
@@ -60,6 +60,8 @@ spec:
               protocol: TCP
             - name: otlp-grpc
               port: 4317
+            - name: otlp-http
```

Comment:
as discussed, this ports section is not needed, if standard receivers used. all those ports will end up in the Service/Ingress without mentioning them here